import "nanopb.proto";
import "header.proto";


message DiscoverMessage
{

}